### ISSUE TITLE
Spacing and organisation of headers on tables

### ISSUE THREAD

**Issue by ekidneyrh** [containers/podman-desktop#7948]
### Is your enhancement related to a problem? Please describe

For the most part, the tables on the various pages are quite similar, with similar headings, yet the spacing of each column it varies depending on which page you're on. 

![table-alignments](https://github.com/containers/podman-desktop/assets/87311656/4889e787-2749-4072-ade0-b45239982ef6)

When switching between pages it can get confusing and takes time for the eyes to readjust. 

### Describe the solution you'd like

It would be great to have the headers spaced out evenly across all the pages. Some names are longer than other, but I'd have them on the same cut-off point. Having the headings in the same order and the variables in the same slot would be great too E.g.:

The heading on each table to read:

| STATUS | NAME | ENVIRONMENT | [VARIABLE] | AGE | ACTIONS

The variables atm are:

    Containers page → IMAGES

    Pods page → CONTAINERS

    Images page → SIZE

    Volumes page → SIZE

**Comment by deboer-tim** [comment]
I gave this a quick try by making three changes:

1. Move Age column to the right in Images and Volumes.
2. Make Name '2fr' and all other columns default ('1fr').
3. Reduce Actions column by 20px to consistent 130px. (reducing wasn't strictly necessary, but Volume Actions was default 1fr before)

The problem with 1 is that Size is right aligned, so you end up with this:
<img width="195" alt="Screenshot 2024-07-04 at 8 15 51 AM" src="https://github.com/containers/podman-desktop/assets/19958075/3395e3b0-3c20-49b6-b1b4-0f0c33179561">
We could increase the spacing between all columns (or get super fancy and make spacing dependent on alignment or something), but both columns were more readable and pretty the other way around.

The problem with 2 is that columns that need more space like Containers > Images are unreadable when the window is narrow:
<img width="195" alt="Screenshot 2024-07-04 at 8 24 16 AM" src="https://github.com/containers/podman-desktop/assets/19958075/ca792dc9-b60d-48f2-b58c-44ed2dc7ce90">
I needed to make the window about 1400px wide in other to read it.

When we get to Kubernetes there are pages that will have a different number of columns as well.

I'm happy to take more screenshots, put this in a draft PR so others can try, or screenshare, but this is feeling like we're losing more in usability than we gain in consistency.
